### DISCUSSION TITLE
Pimcore X on MS Windows

### DISCUSSION THREAD

**Discussion by incoxx** [pimcore/pimcore#10976] [Q&A / Support]
Does anyone have a solution how to get PIMCORE X to run on Microsoft Windows 10 with XAMPP?

It worked fine with PIMCORE 6.

The installation via COMPOSER works. The rest of the setup too.
When I try to open http://localhost or http://localhost/admin "nothing" happens. "This page cannot be reached".
There is no error in the Apache / PHP error log. No message in the dev.log.

I am not sure whether PIMCORE X will work in MS Windows environment at all.

Does anyone have any experience here?

Thank you!
Markus

PS: I KNOW Linux is the better solution, and i did several installations. But our customer want´s ... Windows

**Comment by fashxp** [comment]
We recommend to use docker images ...

**Reply by ruccvcs50x** [reply]
docker is super slow  on windows
